doc/adr/0002-use-component-level-micro-frontends.md

```diff
@@ -0,0 +1,29 @@
+# 2. Use component-level micro frontends
```

Comment:
Can we make clear that some component-level micro frontends will exists, but most of the micro front-ends will be page-level?

Reply:
Maybe should limit it to _just_ the header and footer?

Reply:
Reasonable to require another additional ADR if an unexpected new component comes up.